src/particles/transformation/TO_FIXED_S.H

```diff
@@ -23,20 +23,23 @@ namespace impactx
 {
 namespace transformation
 {
-    struct T2Z
+    struct To_Fixed_S
     {
         using PType = ImpactXParticleContainer::ParticleType;
 
-        /** A T-to-Z transformation
+        /** Transformation of particles from fixed time t to fixed location
+         *  s.  At fixed s, each particle is represented by phase space
```

Comment:
Separates doxygen title line from body:
```suggestion
        /** Transformation of particles from fixed time t to fixed location s.
         *
         * At fixed s, each particle is represented by phase space
```